### ISSUE TITLE
Uncover nakedness

### ISSUE THREAD

**Issue by hmw3** [Door43/ulb-en#1011]
Uncover nakedness

There are many different ways that this phrase is UDB'd in the ULB, "disgrace" and "dishonor" being two of them.

Exod. 20:26; Lev. 18:6ff; 20:11, 17ff; Isa. 47:3; Ezek. 16:36f; 22:10; 23:10, 18, 29

hmw3: If this is made literal, delete 06172 and 01540 from "disgrace" tW page.

**Comment by TomWarren** [comment]
The ULB of EXO 20:26 now reads:
\v 26 You must not go up by steps to my altar, so that you nakedness not be uncovered.'"
The ULB of LEV 18:6 now reads:
\v 6 No one must sleep with any close relative to uncover his nakedness. I am Yahweh.
The ULB of LEV 20:11 now reads:
\v 11 If a man lies with his father's wife, he uncovers his father's nakedness. Both the son and his father's wife must certainly be put to death. Their blood is upon them.
The ULB of LEV 20:17 now reads:
\v 17 If a man sleeps with his sister, a daughter of his father or a daughter of his mother, and he uncovers her nakedness, and she sees his nakedness, it is a shameful thing. They must be cut off from the presence of their people, because he has slept with his sister. He must carry his guilt.
The ULB of ISA 47:3 already reads:
\v 3 Your nakedness will be uncovered, yes, your shame will be seen:
\q1 I will take vengeance and will not spare a man.
The ULB of EZK 16:37 now reads:
\v 37 therefore, behold, I will gather all your lovers that you have met, all those whom you loved and all those whom you hated, and I will gather them against you on every side. I will uncover your nakedness to them so they see all your nakedness.
The ULB of EZK 22:10 now reads:
\v 10 In this city men uncover the nakedness of their father. Within the city they have violated unclean women during their menstrual period.
The ULB of EZK 23:10 now reads:
\v 10 They uncovered her nakedness, took her sons and daughters, killed her with the sword, and she became notorious among other women, so they executed judgment on her.
The ULB of EZK 23:18 now reads:
\v 18 When she displayed her acts of prostitution and uncovered her nakedness, 
\q I turned away from her, just as I had turned away in disgust from her sister.
The ULB of EZK 23:29 now reads;
\v 29 They will deal with you hatefully; they will take all your possessions and abandon you naked and bare, and your nakedness will be uncovered just as when you engaged in promiscuity and prostitutions.

There you go. 

Tom
